### ISSUE TITLE
connecting to server when your steamid is in access file

### ISSUE THREAD

**Issue by rulex** [MinoMino/minqlx#6]
Stuff fails when i had my steam id in my access file.

```
broadcast: print "rulex^7 connected\n"
[minqlx.log_exception] ERROR: Traceback (most recent call last):
[minqlx.log_exception] ERROR:   File "minqlx/minqlx/_handlers.py", line 51, in handle_client_command
[minqlx.log_exception] ERROR:     player = minqlx.Player(client_id)
[minqlx.log_exception] ERROR:   File "minqlx/minqlx/_player.py", line 85, in __init__
[minqlx.log_exception] ERROR:     .format(client_id))
[minqlx.log_exception] ERROR:   File "minqlx/minqlx/_player.py", line 134, in _invalidate
[minqlx.log_exception] ERROR:     raise NonexistentPlayerError(e)
[minqlx.log_exception] ERROR: minqlx._player.NonexistentPlayerError: Tried to initialize a Player instance of nonexistant player 0.
[minqlx.log_exception] ERROR: Traceback (most recent call last):
[minqlx.log_exception] ERROR:   File "minqlx/minqlx/_handlers.py", line 51, in handle_client_command
[minqlx.log_exception] ERROR:     player = minqlx.Player(client_id)
[minqlx.log_exception] ERROR:   File "minqlx/minqlx/_player.py", line 85, in __init__
[minqlx.log_exception] ERROR:     .format(client_id))
[minqlx.log_exception] ERROR:   File "minqlx/minqlx/_player.py", line 134, in _invalidate
[minqlx.log_exception] ERROR:     raise NonexistentPlayerError(e)
[minqlx.log_exception] ERROR: minqlx._player.NonexistentPlayerError: Tried to initialize a Player instance of nonexistant player 0.
```

**Comment by mattiZed** [comment]
I second that.

**Comment by MinoMino** [comment]
Hmm, that's weird. I've always had my ID in my access file, but I've never seen the error. I'll try messing around with the access file and see if I can reproduce it.

**Comment by mattiZed** [comment]
I tried to reproduce it myself yesterday and couldn't - strange that. Maybe it was caused by something else and not related to access.txt!

**Comment by MinoMino** [comment]
The above commit should be useful to try to debug this one. It's included in v0.0.3, so if it happens while you're on v0.0.3 or above, I'd appreciate it if I could have a look at the console output. I can't reproduce it at all.

**Comment by MinoMino** [comment]
Seems like this issue was resolved along with 102faed023c338319bced3cbb03d59db998839e7.
